tools/generator/test/test_languages.py

```diff
@@ -112,3 +116,23 @@ def compile_language(self, language, champion_file_name):
             )
         except subprocess.CalledProcessError as e:
             self.fail("Tester compilation failed with output:\n" + e.output)
+
+
+class CheckLanguages(unittest.TestCase):
```

Comment:
It took me a while to understand what this class is doing, and how it's different from TestLanguages above. Rename to `TestExampleGames` and `def test_example_games(self)`?